x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rule_snooze/panel/base_snooze_panel.tsx

```diff
@@ -213,6 +213,7 @@ export const BaseSnoozePanel: React.FunctionComponent<BaseSnoozePanelProps> = ({
                         style={{
                           paddingLeft: '9px',
                           paddingRight: '9px',
+                          height: '36px',
```

Comment:
Also while I was editing this code I noticed the schedule buttons seemed to have lost their vertical padding:

<img width="462" alt="Screenshot 2023-04-03 at 4 10 55 PM" src="https://user-images.githubusercontent.com/1445834/229535226-42fec6da-f252-4c05-bf7c-1398b81eab5b.png">